dependencies.json

```diff
@@ -0,0 +1,187 @@
+{
+  "pins": {
+    "external": {
+      "numpy": ">= 1.12, < 1.16",
+      "sphinx": ">= 1.4, < 1.8"
+    },
+    "internal": {
+      "pymedphys": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_analysis": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_coordsandscales": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_databases": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_dicom": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_electronfactors": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_fileformats": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_labs": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_logfiles": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_sphinxtheme": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_toolbox": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_utilities": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_workshops": ">= 0.8.0dev2, < 0.9.0",
+      "pymedphys_xlwings": ">= 0.8.0dev2, < 0.9.0"
+    }
+  },
+  "tree": {
+    "pymedphys": {
+      "external": [
+        "jinja2",
+        "notebook"
+      ],
+      "internal": [
+        "pymedphys_analysis",
+        "pymedphys_coordsandscales",
+        "pymedphys_databases",
+        "pymedphys_dicom",
+        "pymedphys_electronfactors",
+        "pymedphys_fileformats",
+        "pymedphys_labs",
+        "pymedphys_logfiles",
+        "pymedphys_toolbox",
+        "pymedphys_utilities",
+        "pymedphys_workshops",
+        "pymedphys_xlwings"
+      ]
+    },
+    "pymedphys_analysis": {
+      "external": [
+        "scipy",
+        "numpy",
+        "pandas",
+        "matplotlib"
+      ],
+      "internal": [
+        "pymedphys_coordsandscales",
+        "pymedphys_utilities",
+        "pymedphys_dicom"
```

Comment:
This dependency tree was written manually, but it was designed such that it should be able to be created automatically with https://github.com/thebjorn/pydeps (in another pull request)

Reply:
Not written manually any more...